### PR TITLE
add the user model

### PR DESCRIPTION
**Pull request by Schparky**

### CODE REVIEW

File: application/migrations/schema.sql [hunk 1]
```diff
@@ -31,13 +31,54 @@ CREATE TABLE public.schema_migration (
 
 ALTER TABLE public.schema_migration OWNER TO "user";
 
+--
```
Comment (Baggerone) [inline application/migrations/schema.sql:34]: We gitignore this file in Cover.  Should we be ignoring it here as well?

**Review by Baggerone — APPROVED** [review]

File: application/migrations/schema.sql (hunk 1 above)
Comment (Schparky) [reply application/migrations/schema.sql:34]: I personally like to see how the database changes in PR reviews, but if you don't like it let me know.

File: application/models/models.go [hunk 2]
```diff
@@ -19,4 +25,86 @@ func init() {
...
+		return err // appErrorFromDB(err, api.ErrorCreateFailure)
+	}
+
+	if valErrs.HasAny() {
+		return valErrs
+		//return api.NewAppError(
+		//	errors.New(flattenPopErrors(valErrs)),
+		//	api.ErrorValidation,
+		//	api.CategoryUser,
+		//)
```
Comment (forevermatt) [inline application/models/models.go:57]: What are the commented-out bits of code for/about?

**Review by forevermatt — COMMENTED** [review]
Added questions here and on schema spreadsheet

File: application/models/models.go (hunk 2 above)
Comment (Schparky) [reply application/models/models.go:57]: I pulled most of this code from Cover, and didn't want to bring in the whole error reporting portion yet.

**Pull request merged**